privacy-contracts/src/main/solidity/PrivacyGroup.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity ^0.5.9;
+import "./PrivacyInterface.sol";
+
+contract PrivacyGroup is PrivacyInterface {
```

Comment:
I think we should give that contract a name that is more descriptive. What about DefaultOnChainPrivacyGroupManagementContract? 
Maybe without the OnChain?

Reply:
fixed